### ISSUE TITLE
Preserve a11y mode for Gabii on Fulcrum

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#1442]
In the Beta version of Gabii on DLXS, if a browser is not compatible with WebGL or a user needs an accessible version of the text, toggling on a11y mode replaces links to Unity with links to the database and closes the 3-D model modal. This feature should be preserved in the Fulcrum version of Gabii.

In this case, you will need to replace the links in the EPUB's `iframe` when toggled on or off.

**Comment by respinos** [comment]
Possible alternative approach --- given you have

```
var click_handler = function(target) { ... };
reader.on('ready:contents', function(contents) {
    var links = contents.content.querySelectorAll("a[data-unity]");
    for(var i =0, n = links.length; i < n; i++) {
       var link = links[i];
       link.addEventListener('click', click_handler);
    }
})
```

to capture the `data-unity` clicks --- your `click_handler` could detect the `a11y` mode and either manipulate the Unity player or open a link to the database.

**Comment by mbakeryo** [comment]
We need tickets 1441, 1396 to be completed first.
